cellxgene_schema_cli/cellxgene_schema/gencode_files/gene_info.yml

```diff
@@ -13,3 +13,6 @@ mouse:
 sars_cov_2:
   description: sars_cov_2
   url: ftp://ftp.ensemblgenomes.org/pub/viruses/gtf/sars_cov_2/Sars_cov_2.ASM985889v3.101.gtf.gz
+fruit_fly:
+  description: fruit_fly
```

Comment:
nit: we use the scientific name for the description for everything else--let's use 'drosophila_melanogaster'